### DISCUSSION TITLE
Virtual Tour plugin

### DISCUSSION THREAD

**Discussion by mistic100** [mistic100/Photo-Sphere-Viewer#599] [Announcements]
A long awaited feature was released in version 4.4.0 : a virtual tour system.

The [Virtual Tour plugin](https://photo-sphere-viewer.js.org/plugins/plugin-virtual-tour.html) allows to define multiple panoramas automatically or manually linked together.

The configuration can be static or loaded from a server; panoramas can be positionned with GPS coordinates; and it can of course be coupled with [markers](https://photo-sphere-viewer.js.org/plugins/plugin-markers.html) and the new [Compass plugin](https://photo-sphere-viewer.js.org/plugins/plugin-compass.html).

**Comment by SequinedAptOrchestra** [comment]
Fantastic addition to the product.  I must say it is well done.  I was able to link some 360 vacation photos together and view on multiple devices and include the gyroscope plugin as well (only works on Surface and not on iPhone but oh well).  Really nice way to get lightweight 360 tours while avoiding most cloud based solutions out there.  

Before I delve too deep in the click event to get coordinates, is there an easier way to configure/create the markers, arrows, and polylines?  I was going to dump into the console and update my plugin code manually.

**Reply by mistic100** [reply]
If you are looking for a GUI to create the markers, no there is not. The click event is indeed what I use to get the coordinates I want for my markers.

```js
viewer.on('click', (e, position) => {
  console.log(position);
});
```

=> #613
